draft-ietf-quic-recovery.md

```diff
@@ -320,10 +320,17 @@ SHOULD be set for twice the newly computed smoothed RTT.
 
 When crypto packets are sent, the sender SHOULD set a timer for the crypto
 timeout period.  Upon timeout, the sender MUST retransmit all unacknowledged
-CRYPTO data by calling RetransmitAllUnackedCryptoData(). On each consecutive
-expiration of the crypto timer without receiving an acknowledgement for a new
-packet, the sender SHOULD double the crypto handshake timeout and set a timer
-for this period.
+CRYPTO data if possible.
+
+Until the server has confirmed the path, it must not send more than 3
+times the number of received bytes, as specified in the {{QUIC-TRANSPORT}}.
```

Comment:
Is this MUST NOT?  You might avoid duplicating the requirement by saying that the server is limited in the amount of data it can send, as specified in {{QUIC-TRANSPORT}}.